lib.php

```diff
@@ -457,7 +457,8 @@ function local_pages_get_fontawesome_icon_map() {
         'local_pages:t/block' => 'fa-ban',
         'local_pages:t/block_to_dock_rtl' => 'fa-chevron-right',
         'local_pages:t/block_to_dock' => 'fa-chevron-left',
-        'local_pages:t/calc_off' => 'fa-calculator', // TODO: Change to better icon once we have stacked icon support or more icons.
+        'local_pages:t/calc_off' => 'fa-calculator',
+        // TODO: Change to better icon once we have stacked icon support or more icons.
```

Comment:
Are you sure this line of comment is belong to ``'local_pages:t/calc' => 'fa-calculator'``?

From what I can see from the previous version the comment is belong to ``'local_pages:t/calc_off' => 'fa-calculator'``

```suggestion
        // TODO: Change to better icon once we have stacked icon support or more icons.
        'local_pages:t/calc_off' => 'fa-calculator',
```